package.json

```diff
@@ -2,16 +2,16 @@
   "name": "@testing-library/angular-app",
   "version": "0.0.0-semantically-released",
   "scripts": {
-    "ng": "ng",
+    "ng": "nx",
     "nx": "nx",
-    "postinstall": "ngcc",
-    "start": "ng serve",
+    "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main",
```

Comment:
Do we still need this?

Reply:
Yes, in a freshly create NX workspace, this is the default.